terraform-modules/k8s/k8s-node-pool.tf

```diff
@@ -18,8 +18,6 @@ resource "google_container_node_pool" "node-pool" {
     auto_upgrade = true
   }
 
-  version = var.k8s_version
```

Comment:
The Google provider docs [say](https://www.terraform.io/docs/providers/google/r/container_node_pool.html#version) that setting `version` when `auto_upgrade` is true is "highly discouraged", because they end up fighting each other.